handler/core/src/test/java/io/knotx/fragments/task/ConfigurationTaskProviderTest.java

```diff
@@ -0,0 +1,324 @@
+/*
```

Comment:
This file is commented. We plan to refactor this part so it will be fixed in next PRs. Step by step.